src/main/java/wolox/training/models/Book.java

```diff
@@ -112,6 +115,7 @@ public String getAuthor() {
     }
 
     public void setAuthor(String author) {
+        Preconditions.checkArgument(Strings.isNullOrEmpty(author), "Author field is required");
```

Comment:
You can call also `checkNotNull` method instead of `checkArgument` to be more readable and write less code. This is also a method of guava if you want you can change it, it is your decision.